src/Platform/Microsoft.Testing.Extensions.AzureDevOps/AzureDevOpsReporter.cs

```diff
@@ -24,6 +24,7 @@ internal sealed class AzureDevOpsReporter :
     private static readonly char[] NewlineCharacters = new char[] { '\r', '\n' };
     private readonly ICommandLineOptions _commandLine;
     private readonly IEnvironment _environment;
+    private readonly string _deterministicBuildRoot = $"/_/";
```

Comment:
- Should it be static/const?
- nit: `$` seems redundant as there are no interpolation holes.